src/Producer.py

```diff
@@ -0,0 +1,79 @@
+"""
+<Program Name>
+  Producer
+
+<Started>
+  July 2019
+
+<Author>
+  Junjie Ge
+
+<Purpose>
+  Instance of posix_omni_parser that poputales the syscall_object and trace in
+  trace manager
+
+"""
+
+import logging
+import sys
+
+from posix_omni_parser.parsers.StraceParser import StraceParser
+
+import consts
+
+class Producer:
+  def __init__(self, trace_file, pickle_file, tm):
+    self.tracefile = trace_file
+    self.trace_manager = tm
+    self.parser = StraceParser(self.tracefile, pickle_file)
+
+  def produce(self, thread_condition):
+    with open(self.tracefile, 'r') as fh:
+      
+      # Finding and ignoring everything before 'syscall_'
+      while True:
+        try:
+          trace_line = fh.next()
+        except StopIteration:
+          logging.debug("Incomplete Trace. Trace ended without 'syscall_'")          
+          sys.exit(1)
```

Comment:
We don't want to exit here.  We should re-rase a RuntimeError or something like that.  Actually exiting if appropriate is something to do at a higher level

Reply:
Done